app/models/match.rb

```diff
@@ -70,6 +70,61 @@ def confirm!
     save!
   end
 
+  def find_other_available_match_for_user
+    return nil if confirmed?
```

Comment:
`nil` is implicit when calling `return` without an argument

```suggestion
    return if confirmed?
```